server/controllers/documentController.js

```diff
@@ -1,132 +1,298 @@
+      }]
+    };
+    Users.findById(request.params.id, queryBuilder)
+      .then((results) => {
+        if (results) {
+          const result = results.dataValues.documents;
+          const searchResult = [];
+          result.forEach((document) => {
+            if (userRole === 1) {
+              searchResult.push(document.dataValues);
+            } else if ((document.access === 'public' ||
+              document.user.dataValues.RoleId === userRole)
+              && document.access !== 'private') {
+              searchResult.push(document.dataValues);
+            } else if (document.access === 'private' && document.OwnerId === Owner) {
+              searchResult.push(document.dataValues);
+            }
+          });
+          response.status(200).send({
+            success: true,
+            message: `documents belonging to user id: ${request.params.id} found`,
+            documents: searchResult
+          });
+        } else {
+          response.status(404).send({
+            success: false,
+            message: `No document found for this user as User with id ${request.params.id} does not exist`
+          });
+        }
+      })
+      .catch((error) => {
+        response.status(401).send({
+          success: false,
+          message: error.message
+        });
+      });
+  }
+  /**
+   * Edit and Update User documents in the database
+   * Users only have access to their own documents
+   * @param{Object} request - Server request
+   * @param{Object} response - Server response
+   * @return {Void} - returns Void
+   */
+  static updateDocument(request, response) {
+    const Owner = request.decoded.UserId;
+    const Role = request.decoded.RoleId;
+    Documents.findOne({
+      where: {
+        id: request.params.id
+      }
+    })
+      .then((document) => {
+        if (document.OwnerId === Owner || Role === 1) {
+          document.update(request.body)
+            .then(updatedDocument => response.status(201).send(updatedDocument))
             .catch(error => response.status(401).send(error));
-    }
-    /**
-     * 
-     */
-    static updateDocument(request, response) {
-        Documents.findOne({ where: { id: request.body.id } })
-            .then(document => {
-                if (document) {
-                    document.update(request.body)
-                        .then(updatedDocument => response.status(201).send(updatedDocument))
-                        .catch(error => response.status(401).send(error));
-                } else {
-                    response.status(404).send({
-                        success: false,
-                        message: 'Document not found'
-                    })
-                }
-            })
+        } else {
+          response.status(403).send({
+            success: false,
+            role: Role,
+            message: 'Forbidden, You are not authorized tos update this document as it doesn\'t belong to you'
```

Comment:
Typo here and use a `""` for your string if you need to use an apostrophe instead of escaping it. Also dito messages comment above.